### ISSUE TITLE
Q4Q4 post-train Resnet-50 cannot get accuracy 73.3.

### ISSUE THREAD

**Issue by haichaoyu** [submission2019/cnn-quantization#11]
Hello,

Thanks for the great codebase. I tried `Experiment W4A4 + ACIQ + Bit Alloc(A) + Bit Alloc(W) + Bias correction` by
`python inference/inference_sim.py -a resnet50 -b 512 -pcq_w -pcq_a -sh --qtype int4 -qw int4 -c laplace -baa -baw -bcw
`
but only get accuracy 71.6. Is there any particular setting that should be followed? Thanks.

My environment:
Pytorch 1.3
Torchvision 0.4.2

**Comment by haichaoyu** [comment]
Hi @ynahshan, any advice? Thanks.

**Comment by nfrumkin** [comment]
I got the same results here. Reproducing using explicit commands:
4A4W naive: 63.68%
4A4W ACIQ + Bias Correction + BitAlloc (weights + acts): 71.72%
